### PR TITLE
Custom applications new theme release note

### PR DESCRIPTION
**Pull request by CarlosCortizasCT**
#### Summary

Added the release note for Custom Applications developers about the new theme for the Merchant Center.

**IMPORTANT**: This PR depends on #3057

### CODE REVIEW

File: website/src/releases/2023-05/new-theme.mdx [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+date: 2023-05-09
+title: New UI styles for Merchant Center
+description: The Merchant Center styles have been updated.
+type: announcement
+topics:
+  - Merchant Center
+  - Development
+---
+
+Now that we've updated the Merchant Center to have a modern, cleaner and better structured UI, you can also make your Custom Application to use these same styles.
```
[the remark this answers is not in the document]
Comment (mmaltsev-ct) [reply website/src/releases/2023-05/new-theme.mdx:11]: How about:
`Now that we've redesigned the Merchant Center to enhance user experience and usability, you can also make your Custom Application look consistent with the rest of the Merchant Center.`?

File: website/src/releases/2023-05/new-theme.mdx [hunk 2]
```diff
@@ -0,0 +1,18 @@
+---
+date: 2023-05-09
+title: New UI styles for Merchant Center
```
Comment (naveenchkrvrthy) [inline website/src/releases/2023-05/new-theme.mdx:3]:
```suggestion
title: New UI style for the Merchant Center
```

File: website/src/releases/2023-05/new-theme.mdx [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+description: The Merchant Center styles have been updated.
```
Comment (naveenchkrvrthy) [inline website/src/releases/2023-05/new-theme.mdx:4]:
```suggestion
description: The UI style of the Merchant Center is now updated, following its redesign.
```

File: website/src/releases/2023-05/new-theme.mdx [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+---
+
+Now that we've redesigned the Merchant Center to enhance user experience and usability, you can also make your Custom Application look consistent with the rest of the Merchant Center.
+You'll just need to update your `ui-kit` and `app-kit` dependencies to the latest major versions following the instructions given in our [previous announcement](./2023-04-21-custom-applications-v22-and-ui-kit-v16).
+
+We don't expect you to need to do any extra work regarding the styles of your components.
+
+One important feature to mention about the lastest `app-kit` version is that it now includes a new set of components with the goal of helping you laying out the content or your views and reuse the same pattern we use in the Merchant Center default applications.
+You can read more about them in [this section](../development/page-content-layouts) of the docs.
+
```
[the remark this answers is not in the document]
Comment (CarlosCortizasCT) [reply website/src/releases/2023-05/new-theme.mdx:18]: Hi @naveenchkrvrthy 

Thanks for your review and suggestion.

I'm worried your proposal does not convey the main message we would need to share with Custom Application developers which is they should update their `ui-kit` and `app-kit` dependencies.
With your proposal it seems the main message is about the page content layout components and also the sentence about don't needing any extra work feels weird as it's not related to those components but to the process of updating a Custom Application.

I think the priority of the messages should be kept:
1. Merchant Center UI has been redesigned
2. You need to update `ui-kit` and `app-kit` dependencies in your Custom Application
3. We don't expect you to require any extra work to adapt your Custom Application to the new design
4. On top of that, now we provide a set of components to help when laying out pages content

File: website/src/releases/2023-05/new-theme.mdx [hunk 5]
```diff
@@ -0,0 +1,18 @@
...
+title: New UI styles for Merchant Center
+description: The Merchant Center styles have been updated.
+type: announcement
+topics:
+  - Merchant Center
+  - Development
+---
+
+Now that we've redesigned the Merchant Center to enhance user experience and usability, you can also make your Custom Application look consistent with the rest of the Merchant Center.
+You'll just need to update your `ui-kit` and `app-kit` dependencies (use the latest version if possible).
```
Comment (CarlosCortizasCT) [inline website/src/releases/2023-05/new-theme.mdx:12]: @emmenko My main concern here is that we're not telling developers which version they should use for their Custom Applications to look like the rest of the Merchant Center.
If I was such a developer, I would be wondering: "Ok, if I don't use the latest versions, what are the minimum versions I should use to take full advantage of the new design?"

File: website/src/releases/2023-05/new-theme.mdx [hunk 6]
```diff
@@ -0,0 +1,18 @@
...
+
+We don't expect you to need to do any extra work regarding the styles of your components.
+
+One important feature to mention about the lastest `app-kit` version is that it now includes a new set of components with the goal of helping you laying out the content or your views and reuse the same pattern we use in the Merchant Center default applications.
+You can read more about them in [this section](../development/page-content-layouts) of the docs.
```
Comment (emmenko) [inline website/src/releases/2023-05/new-theme.mdx:17]:
```suggestion
Additionally, as part of the new Merchant Center design, we introduced new [layout guidelines for displaying content on a page](/development/page-content-layouts), including new UI components to implement these layouts.
We recommend for Custom Applications to start following these guidelines and use the page content layout components.
```

File: website/src/releases/2023-05/new-theme.mdx (hunk 2 above)
Comment (CarlosCortizasCT) [reply website/src/releases/2023-05/new-theme.mdx]: Updated here: f7e78ed415646fc7393db468b61859a8790fc0ee

File: website/src/releases/2023-05/new-theme.mdx (hunk 3 above)
Comment (CarlosCortizasCT) [reply website/src/releases/2023-05/new-theme.mdx]: Updated here: f7e78ed415646fc7393db468b61859a8790fc0ee

File: website/src/releases/2023-05/new-theme.mdx (hunk 5 above)
Comment (CarlosCortizasCT) [reply website/src/releases/2023-05/new-theme.mdx:12]: Updated here: f7e78ed415646fc7393db468b61859a8790fc0ee

File: website/src/releases/2023-05/new-theme.mdx (hunk 6 above)
[the remark this answers is not in the document]
Comment (CarlosCortizasCT) [reply website/src/releases/2023-05/new-theme.mdx]: Updated here: f7e78ed415646fc7393db468b61859a8790fc0ee

Comment (CarlosCortizasCT) [reply website/src/releases/2023-05/new-theme.mdx]: Updated here: f7e78ed415646fc7393db468b61859a8790fc0ee

**Review by emmenko — APPROVED** [review]
Thanks Carlos, this looks good to me now 👌

File: website/src/releases/2023-05/new-theme.mdx [hunk 7]
```diff
@@ -0,0 +1,19 @@
...
+
+Now that we've redesigned the Merchant Center to enhance user experience and usability, you can also make your Custom Application look consistent with the rest of the Merchant Center.
+You'll just need to update your `ui-kit` and `app-kit` dependencies (use the latest version if possible).
+
+We don't expect you to need to do any extra work regarding the styles of your components.
+
+Additionally, as part of the new Merchant Center design, we introduced new [layout guidelines for displaying content on a page](/development/page-content-layouts), including new UI components to implement these layouts.
+We recommend for Custom Applications to start following these guidelines and use the page content layout components.
+
+As always, if you have questions or feedback you can open a [GitHub Discussion](https://github.com/commercetools/merchant-center-application-kit/discussions) or a [GitHub Issue](https://github.com/commercetools/merchant-center-application-kit/issues).
```
Comment (naveenchkrvrthy) [inline website/src/releases/2023-05/new-theme.mdx:19]: @CarlosCortizasCT I've made minimal edits keeping in mind the priority of the message. Ok now?
```suggestion
You can now get your Custom Application to resemble the default applications in the redesigned Merchant Center. To achieve this, update the `ui-kit` and `app-kit` dependencies to the latest version. We don't expect you to do any extra work for the style of your components.

The new Merchant Center design also brings new [layout guidelines](/development/page-content-layouts) to display page content, and new UI components to implement the layouts. We recommend following these guidelines and using the layout components for your Custom Applications.

If you have questions or feedback, open a GitHub [discussion](https://github.com/commercetools/merchant-center-application-kit/discussions) or [issue](https://github.com/commercetools/merchant-center-application-kit/issues).
```

**Review by mmaltsev-ct — APPROVED** [review]
Looks great, thank you!

File: website/src/releases/2023-05/new-theme.mdx [hunk 8]
```diff
@@ -0,0 +1,18 @@
...
+  - Merchant Center
+  - Development
+---
+
+Now that we've updated the Merchant Center to have a modern, cleaner and better structured UI, you can also make your Custom Application to use these same styles.
+You'll just need to update your `ui-kit` and `app-kit` dependencies to the latest major versions following the instructions given in our [previous announcement](./2023-04-21-custom-applications-v22-and-ui-kit-v16).
+
+We don't expect you to need to do any extra work regarding the styles of your components.
+
+One important feature to mention about the lastest `app-kit` version is that it now includes a new set of components with the goal of helping you laying out the content or your views and reuse the same pattern we use in the Merchant Center default applications.
```
[the remark this answers is not in the document]
Comment (naveenchkrvrthy) [reply website/src/releases/2023-05/new-theme.mdx]: @mmaltsev-ct The above suggestion is respective to the initial draft of the content, while meanwhile the content was updated. This is the the current one:

>The new Merchant Center design also brings new [layout guidelines](/development/page-content-layouts) to display page content, and new UI components to implement the layouts. We recommend following these guidelines and using the layout components for your Custom Applications.

Does it still need to be modified?

**Review by naveenchkrvrthy — APPROVED** [review]

File: website/src/releases/2023-05/new-theme.mdx (hunk 8 above)
[the remark this answers is not in the document]
Comment (mmaltsev-ct) [reply website/src/releases/2023-05/new-theme.mdx]: I think it looks good @naveenchkrvrthy, thanks for checking!

**Pull request merged**